ref.go

```diff
@@ -25,9 +25,12 @@ import (
 
 // used to ref object,list,map
 type _refElem struct {
-	// record the kind of target, objects are the same only if the address and kind are the same
+	// different kind may share the same address, i.e. slice and its first element
```

Comment:
the previous comment should not be deleted

Reply:
Ok, I will revert this comment.

Reply:
I have add the previous comment back:

https://github.com/dubbogo/hessian2/blob/ad9cb8199f7324a978de9d46309d13f92ae5b070/ref.go#L27-L29